lib/impl/skia/font.cpp

```diff
@@ -174,20 +170,12 @@ namespace cycfi::artist
                std::string key = (const char*) family;
                trim(key);
 
-               if (auto it = font_map().find(key); it != font_map().end())
-               {
-                  it->second.push_back(entry);
-               }
-               else
-               {
-                  font_map()[key] = {};
-                  font_map()[key].push_back(entry);
-               }
+               font_map[key].push_back(std::move(entry));
             }
          }
       }
 
-      font_entry const* match(font_descr descr)
+      font_entry* match(font_map_type& font_map, font_descr descr)
```

Comment:
Why is this un-consted?

Reply:
Ah, I see... because you have to place some cache info, correct?

Reply:
OK, I get it. No need to answer. I got my answer after understanding what's happening. I'll merge now (again) :-)